phoenicis-tools/src/test/java/org/phoenicis/tools/archive/ExtractorTest.java

```diff
@@ -83,7 +89,12 @@ public void testUncompress_withSymbolicLinks() throws IOException {
     }
```

Comment:
Mh it's only in the test. I don't think it makes sense to throw in some tests and try-catch in others.